src/components/backend-ai-settings-view.ts

```diff
@@ -433,27 +551,202 @@ export default class BackendAiSettingsView extends BackendAIPage {
+      if (this.selectedSchedulerType === 'fifo' || (this.selectedSchedulerType !== 'fifo' && tempNumRetries === '0')) {
+        // handle scheduler options
+        const numRetries = parseInt(tempNumRetries).toString();
+        const options = {
+          'num_retries_to_skip': numRetries
+        };
+        globalThis.backendaiclient.setting.set(`plugins/scheduler/${this.selectedSchedulerType}`, options)
+        .then((response) => {
+          this.notification.text = _text('notification.SuccessfullyUpdated');
+          this.notification.show();
+          this.options['scheduler'] = this.selectedSchedulerType;
+          this.schedulerOptions = {...this.schedulerOptions, ...options};
+          this.update(this.options);
+          this.update(this.schedulerOptions);
+          this._hideEnvDialog();
+        })
+        .catch((err) => {
+          this.notification.text = PainKiller.relieve('Couldn\'t update scheduler setting.');
+          this.notification.detail = err;
+          this.notification.show(true, err);
+        })
+      } else if (tempNumRetries !== '0') {
+        this.notification.text = _text('settings.FifoOnly');
         this.notification.show();
-        // console.log(response);
-      }).catch((err) => {
-        this.notification.text = PainKiller.relieve('Couldn\'t update scheduler setting.');
-        this.notification.detail = err;
-        this.notification.show(true, err);
-      });
+        this.shadowRoot.querySelector('#num-retries').value = '0';
+      }
     }
   }
+
+  /**
+   * Change this.selectedSchedulerType value and the scheduler options
+   * 
+   * @param {HTMLElement} e - scheduler setting component
+   */
+  changeSelectedScheduleType(e) {
+    this.selectedSchedulerType = e.target.value;
+    this.updateSettings();
+    for (const [key] of Object.entries(this.schedulerOptions)) {
+      globalThis.backendaiclient.setting.get(`plugins/scheduler/${this.selectedSchedulerType}/${key}`).then((response) => {
+        if (response['result'] === null) {
+          this.shadowRoot.querySelector('#' + this._findIdByOption(key)).value = '0';
+        } else {
+          this.shadowRoot.querySelector('#' + this._findIdByOption(key)).value = response['result'];
+        }
+      })
+    }
+  }
+
+  /**
+   * Check validation of input.
+   *
+   * @param {Event} e - Dispatches from the native input event each time the input changes.
+   */
+   _validateInput(e) {
+    const textfield = e.target.closest('mwc-textfield');
+    const clamp = (value: number, min: number, max: number) => {
+      return Math.max(min, Math.min(value, max));
+    };
+    if (textfield.value) {
+      textfield.value = Math.round(textfield.value);
+      textfield.value = clamp(textfield.value, textfield.min, textfield.max);
+    }
+  }
+
+  /**
+   * customize validation message.
+   * 
+   * @param {Event}  e - Dispatches from the native input event when input event occurs.
+   */
+  _customizeValidationMessage(e) {
```

Comment:
@Sujin-Kim1
According to[ mwc-textfield documentation](https://github.com/material-components/material-components-web-components/tree/master/packages/textfield) and [ValidityState in native input field](https://developer.mozilla.org/en-US/docs/Web/API/ValidityState), we can customize validation message per each invalid input.

Currently, there are two cases of invalid input.
The first case is empty input, so we can simply show the message such as 'This input is required'.
And the other case is that the input value exceeds[ the minimum or max range in the input field](https://github.com/lablup/backend.ai-webui/pull/980/files#diff-fc606a4cc01af6722e8fc5a686d3db2e7113862c172b2e7d80c5e27659e6503fR405-R406).
In this situation, we can show the message like this, 'please enter value between min value to max value.'